shared/src/main/scala/FuncHelpers.scala

```diff
@@ -10,24 +10,51 @@ object FuncHelpers:
       fn.arity match
         case 1 =>
           val a = ctx.pop()
-          Monad.vectoriseNoFill { a =>
+          VList(ListHelpers.makeIterable(a).map { a =>
             Interpreter.executeFn(fn, args = Some(List(a)))
-          }(a)
+          }*)
```

Comment:
I can't make a suggestion here, but this can be
```scala
ListHelpers.makeIterable(a).vmap { a =>
  Interpreter.executeFn(fn, args = Some(List(a)))
}
```